doc/user_guide/ensembleModel.tex

```diff
@@ -0,0 +1,229 @@
+combine them by defining an \xmlNode{EnsembleModel}:
+\xmlExample{framework/user_guide/EnsembleModel/basic.xml}{Models}
+The first model we will look at is the definition of the \xmlString{ballistics} model.  We note that is uses
+the \xmlString{GenericCode} subType, so it will use the interface as described in the user manual, where
+wildcards in the input file are replaced with RAVEN's sampled values.  The \xmlNode{clargs} (command line
+arguments) include the \xmlString{python} command, the \xmlString{-i} flag to specify the input file, and the
+\xmlString{-o} flag to specify the output file.  These nodes are specific to the Generic code interface and
+will not apply to all codes or models. The \xmlString{Impact} model is defined in a similar manner.
+
+With both Models defined, we can construct the \xmlNode{EnsembleModel}, which we've named
+\xmlString{ballistic\_and\_impact} to describe the codes that are paired (you can name this whatever you
+want).  There is no \xmlAttr{subType} for the \xmlNode{EnsembleModel} currently, so we leave it blank.
+
+Within the \xmlNode{EnsembleModel} node there are two \xmlNode{Model} nodes listed, which will inform the
+Ensemble Model which models need coupling.  The order listed does not matter; the Ensemble Model will sort out
+the order based on the \xmlNode{TargetEvaluation} data objects of each model.
+
+The first \xmlNode{Model} we list within the \xmlNode{EnsembleModel} is the \texttt{ballistics} model.  Note
+that the \xmlAttr{class} and \xmlAttr{type} are used along with the name to find the right model.  Note also
+the name of the model goes in the \emph{text} of the \xmlNode{Model} node, right after the first closing angle
+bracket.  The subnodes of the \xmlString{ballistics} model are the \xmlNode{Input}, which points to the input
+template input file we defined in the \xmlNode{Files} block; and the \xmlNode{TargetEvaluation}, which points
+to the \xmlNode{PointSet} we defined in the \xmlNode{DataObjects} block.  These are critical to allowing the
+Ensemble Model both to run the \xmlString{ballistics} model correctly, as well as the entire ensemble in a
+sensible manner.
+
+In general, some models (such as a \xmlNode{ROM} or \xmlNode{ExternalModel}) do not require any files as
+inputs.  In this case, the \xmlNode{Input} for an input-less model should be a placeholder data object,
+usually with the same input space as the \xmlNode{TargetEvaluation} data object, and a placeholder output.
+For more information, see examples running external models in these guides as well as the user manual.
+
+After the \xmlString{ballistics} model, we see a similar structure pointing to the \xmlString{impact} model.
+The only differences between this model specification and the ballistics model specificatoin are the names of
+the \xmlNode{Input} file and \xmlNode{TargetEvaluation} data object.
+
+That's it!  With the two models defined, and the Ensemble Model assembled, no more work is required to
```

Comment:
```That's it!``` probably we should avoid "excitement" expressions 😸

Reply:
I understand we want to put people in a coma in the user manual, but do we need to use high-level technical writing in our user guide, too?  Removed the interjection for this case.